daprdocs/content/en/developing-applications/building-blocks/service-invocation/service-invocation-overview.md

```diff
@@ -39,22 +43,16 @@ The diagram below is an overview of how Dapr's service invocation works.
 ## Features
 Service invocation provides several features to make it easy for you to call methods between applications.
 
-### Namespace scoping
-
-Applications can be scoped to namespaces for deployment and security, and you can call between services deployed to different namespaces. For more information, read the [Service invocation across namespaces]({{< ref "service-invocation-namespaces.md" >}}) article.
+### Proxying
+- **HTTP proxying**: If you're already using HTTP protocols in your application, using the Dapr HTTP header might be the easiest way to get started. You don't need to change your existing endpoint URLs; just add the `dapr-app-id` header and you're ready to go. For more information, see [Invoke Services using HTTP]({{< ref howto-invoke-discover-services.md >}}). 
```

Comment:
I would not called this the Dapr HTTP Header it implies this is different from HTTP header. And this is also the mechanism, in details. I would say more along on the lines of using existing HTTP Client libraries and then including the Dapr application ID in the the HTTP header calls.

Reply:
Great feedback. I've made these changes in the doc as well